### ISSUE TITLE
Watchdog exception when breaking very large trees.

### ISSUE THREAD

**Issue by jessejohn01** [laynebritton/jl-fast-treecapitator#5]
![image](https://user-images.githubusercontent.com/26558605/220457907-27f731d7-0619-4e16-8b60-0dc72e188569.png)

This exception occurs when capping very large trees. Specifically 2x2 large spruce trees on the server.  Most likely just hitting the watchdog limit on time passed for the calculations.

**Comment by laynebritton** [comment]
A quick fix for this is lowering the `max_depth`, although some testing needs to be done to find the right amount.

Also, I wonder if moving away from the run command async tool and using the fillBlock api helps with run time. Probably not for this case.

**Comment by jessejohn01** [comment]
https://wiki.bedrock.dev/scripting/script-watchdog.html
I was able to determine this is a dedicated server only issue. I disabled the watchdog on the server. I do recommend we monitor for watchdog exceptions and continue to improve performance. I would say the next step is to directly use the fillBlock API command. 

We do have another issue open to handle exceptions better in the future. Currently on a dedicated server, if you attempt to cap a tree and it would normally cause a watch dog error it will not break any further blocks.
